simapp/simd/cmd/root.go

```diff
@@ -97,7 +98,8 @@ func initRootCmd(rootCmd *cobra.Command, encodingConfig params.EncodingConfig) {
 		debug.Cmd(),
 	)
 
-	server.AddCommands(rootCmd, simapp.DefaultNodeHome, newApp, createSimappAndExport)
+	server.
+		AddCommands(rootCmd, simapp.DefaultNodeHome, newApp, createSimappAndExport, addModuleInitFlags)
```

Comment:
nit

```suggestion
	server.AddCommands(rootCmd, simapp.DefaultNodeHome, newApp, createSimappAndExport, addModuleInitFlags)
```

Reply:
yeah, I think it was in a new line before, so I kept that "style"